src/TheThingsNetwork.cpp

```diff
@@ -284,8 +284,9 @@ int TheThingsNetwork::sendBytes(const byte* payload, int length, int port, bool
 
   String response = "";
   if (confirm) {
-    while ((response = readLine(10000)) == "")
+    while ((response = readLine(10000)) == "") {
       delay(1000);
+    }
```

Comment:
No need for 10 s if using indefinitely, and no need to wait for nothing for 1 s;

`while ((response = readLine()) == "");`